tests/integration-tests/conversations/test_concurrent_stream_timing.py

```
@@ -36,33 +36,43 @@
 SERIAL_WALL_SECS = WAIT_SECS * CONCURRENT_REQUESTS
 MAX_WALL_SECS = SERIAL_WALL_SECS * 0.9
 
-# Module where asyncio.to_thread is used in production (router calls start_celery_task_and_stream via to_thread)
-CONVERSATIONS_ROUTER_MODULE = "app.modules.conversations.conversations_router"
-
-
 @pytest.fixture
-def slow_redis_stream_manager(monkeypatch):
-    """RedisStreamManager mock whose wait_for_task_start sleeps WAIT_SECS then returns True."""
-    mock_manager = MagicMock(spec=RedisStreamManager)
+def slow_redis_stream_manager(app, monkeypatch, mock_redis_stream_manager):
```

Ruff (0.16.0)

[warning] Unused function argument: `mock_redis_stream_manager`

(ARG001)